sigopt_sklearn/search.py

```diff
@@ -331,20 +338,25 @@ def _fit(self, X, y, parameter_iterable=None):
                 for sidx, suggestion in enumerate(suggestions):
                     out_sidx = n_folds * sidx
                     scores = [o[0] for o in out[out_sidx:(out_sidx+n_folds)]]
-                    conn.experiments(self.experiment.id).observations().create(
-                        suggestion=suggestion.id,
-                        value=numpy.mean(scores),
-                        value_stddev=numpy.std(scores)
-                    )
+                    self.sigopt_connection \
+                        .experiments(self.experiment.id) \
+                        .observations() \
+                        .create(suggestion=suggestion.id,
+                                value=numpy.mean(scores),
+                                value_stddev=numpy.std(scores))
```

Comment:
Ah, ok. I put in slashes in order to keep this line from running past the 80 character mark. Would

```python
self.sigopt_connection.experiments(self.experiment.id).observations().create(
  arg1=val1,
  arg2=val2,
)
```

or 

```python
(self.sigopt_connection
     .experiments(self.experiment.id)
     .observations()
     .create(
         arg1=val1,
         arg2=val2))
```

be preferred?

On a side note, do we have linter set up?